src/components/message/message-card/MessageCardTools.tsx

```diff
@@ -172,22 +160,16 @@ const MessageCardTools = ({
 					(messageViewType === MessageViewType.JSON ||
 						messageViewType === MessageViewType.FORMATTED) && (
 						<div className='message-card-tools__controls-group'>
-							{['body', 'fields'].map(copyOption => (
+							{['fields'].map(copyOption => (
```

Comment:
we don't need body copying anymore? if that's so array with 1 item looks weird we should refactor that